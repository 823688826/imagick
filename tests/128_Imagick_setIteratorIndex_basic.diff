001+ Fatal error: Uncaught exception 'ImagickException' with message 'Unable to set iterator index' in /home/github/mkoppanen-imagick/tests/128_Imagick_setIteratorIndex_basic.php:8
001- Ok
002+ Stack trace:
003+ #0 /home/github/mkoppanen-imagick/tests/128_Imagick_setIteratorIndex_basic.php(8): Imagick->setiteratorindex(3)
004+ #1 /home/github/mkoppanen-imagick/tests/128_Imagick_setIteratorIndex_basic.php(21): setIteratorIndex(3)
005+ #2 {main}
006+   thrown in /home/github/mkoppanen-imagick/tests/128_Imagick_setIteratorIndex_basic.php on line 8