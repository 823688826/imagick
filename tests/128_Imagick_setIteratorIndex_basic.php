<?php

$firstLayer = 3;

    function setIteratorIndex($firstLayer) {
        $imagick = new \Imagick(realpath("images/LayerTest.psd"));
        $output = new \Imagick();
        $imagick->setIteratorIndex($firstLayer);

        do {
            $output->addImage($imagick->getimage());
        } while($imagick->nextImage());

        $merged = $output->mergeImageLayers(\Imagick::LAYERMETHOD_MERGE);

        $merged->setImageFormat('png');
    $bytes = $merged->getImageBlob();
    if (strlen($bytes) <= 0) { echo "Failed to generate image.";} 
    }

setIteratorIndex($firstLayer) ;
echo "Ok";
?>
