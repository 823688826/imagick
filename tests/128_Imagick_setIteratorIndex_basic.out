Fatal error: Uncaught exception 'ImagickException' with message 'Unable to set iterator index' in /home/github/mkoppanen-imagick/tests/128_Imagick_setIteratorIndex_basic.php:8
Stack trace:
#0 /home/github/mkoppanen-imagick/tests/128_Imagick_setIteratorIndex_basic.php(8): Imagick->setiteratorindex(3)
#1 /home/github/mkoppanen-imagick/tests/128_Imagick_setIteratorIndex_basic.php(21): setIteratorIndex(3)
#2 {main}
  thrown in /home/github/mkoppanen-imagick/tests/128_Imagick_setIteratorIndex_basic.php on line 8